sensor_msgs/CameraInfo cam_info
sensor_msgs/Image depth_image
string[] labels
int64[] x_s
int64[] y_s
---
geometry_msgs/PoseStamped obj_poses